### PR TITLE
Add OIDC login

### LINKED ISSUE #4: OIDC Login

### PR DESCRIPTION
**Pull request by T4rikA**
closes #4 

- [x]  Reminder: Add changes in env file to README as soon as #27 is merged

### CODE REVIEW

File: myhpi/templates/login.html [hunk 1]
```diff
@@ -0,0 +1,9 @@
+{% if user.is_authenticated %}
```
Comment (Paula-Kli) [inline myhpi/templates/login.html:1]: what does `is_authenticated` mean? for me it reads like: "I can login if I am already authenticated" ?

**Comment by Paula-Kli** [comment]
As much as I understand it it looks fine to me :+1:

File: myhpi/templates/login.html (hunk 1 above)
Comment (T4rikA) [reply myhpi/templates/login.html:1]: Exactly, but this is a really good catch because the template is not used yet 😅

Comment (Paula-Kli) [reply myhpi/templates/login.html:1]: what is it for then?

Comment (T4rikA) [reply myhpi/templates/login.html:1]: It should be used, we just forgot to finish this

**Review by jeriox — APPROVED** [review]

**Pull request merged**